packages/select/src/MenuButton.tsx

```diff
@@ -36,33 +65,87 @@ const menuButtonModeOverrides: Record<Mode, string> = {
       }
     }
   `,
-  [Mode.Dark]: css`
-    border-color: transparent;
+  [Theme.Dark]: css`
+    border-color: ${palette.gray.base};
+    background-color: ${palette.gray.dark4};
+    color: ${palette.gray.light3};
+
+    // Override button default color
+    > *:last-child {
+      > svg {
+        color: ${palette.gray.light1};
+      }
+    }
+
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: ${palette.gray.dark4};
+      color: ${palette.gray.light3};
+    }
   `,
 };
 
-const menuButtonDeselectedStyles: Record<Mode, string> = {
-  [Mode.Light]: css`
-    color: ${colorSets['light'].text.deselected};
+// Override default button focus styles
+const menuButtonFocusStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    &:focus {
+      box-shadow: ${focusRing['light'].input};
+      border-color: rgba(255, 255, 255, 0);
+    }
   `,
-  [Mode.Dark]: css`
-    color: ${colorSets['dark'].text.deselected};
+  [Theme.Dark]: css`
+    &:focus {
+      background-color: ${palette.gray.dark4};
+      box-shadow: ${focusRing['dark'].input};
+      border-color: rgba(255, 255, 255, 0);
```

Comment:
Just curious - is there a reason this is preferred to the keyword `transparent`?

Reply:
Yeah, there is a note in the `icon-button` component -- Safari treats "transparent" values in CSS as transparent black instead of white which can make things render differently across browsers if not defined explicitly. So I stick to rgba.